### PR TITLE
[Ranks] Remove hardcoded config calls

### LINKED ISSUE #427: [Ranks] Remove hardcoded config calls
#404.

### PR DESCRIPTION
**Pull request by Injabie3**
This PR closes #427 by doing the following:
- Add KEY_* constants.
- Use the KEY_* constants with get_attr method calls.

There are no changes to existing behaviour.

### CODE REVIEW

File: cogs/ranks/ranks.py [hunk 1]
```diff
@@ -222,7 +222,7 @@ async def _settingsMaxpoints(self, ctx: Context, maxPoints: int = 25):
             )
             return
 
-        await self.config.guild(ctx.guild).maxPoints.set(maxPoints)
+        await self.config.guild(ctx.guild).get_attr(KEY_MAX_POINTS).set(maxPoints)
 
         await ctx.send(
             ":white_check_mark: **Ranks - Max Points**: Users can gain "
```
Comment (quachtridat) [inline cogs/ranks/ranks.py:228]:
```py3
            ":white_check_mark: **Ranks - Max Points**: Users can gain "
            f"up to {maxPoiunts} points per eligible message."
```
Typo.

Comment (Injabie3) [reply cogs/ranks/ranks.py:228]: omg LOL

Comment (Injabie3) [reply cogs/ranks/ranks.py:228]: thanks, fixed now 🙏

**Review by quachtridat — APPROVED** [review]
Changes LGTM.

**Pull request merged**